streams/src/test/java/org/apache/kafka/streams/integration/KStreamAggregationIntegrationTest.java

```diff
@@ -327,7 +329,7 @@ public void shouldAggregateWindowed() throws Exception {
         startStreams();
 
         final List<KeyValueTimestamp<Windowed<String>, Integer>> windowedMessages = receiveMessagesWithTimestamp(
-            new TimeWindowedDeserializer<>(),
```

Comment:
I don't think we should get rid of the generics unless we have to

Reply:
The window size has to be set either in the constructor or in the configs so if we use the generics we have to use the configs, but the general idea was to push users to do the constructor over the configs. Also in this scenario, I wanted to confirm that it would work fine without configs

Reply:
That is fine. but I think It should be like `new TimeWindowedDeserializer<String>(windowedSerde.deserializer(), 500L)` if that complies. We want to keep the type checking